tools/benchmark/src/MochaReporter.ts

```diff
@@ -26,21 +26,21 @@ import { BenchmarkData, BenchmarkResult } from "./runBenchmark";
  *
  * See https://mochajs.org/api/tutorial-custom-reporter.html for more information about custom mocha reporters.
  */
-// eslint-disable-next-line @typescript-eslint/no-extraneous-class, unicorn/prefer-module
+// eslint-disable-next-line unicorn/prefer-module
 module.exports = class {
+	public data: Map<Test, BenchmarkResult> = new Map();
```

Comment:
If this never gets reassigned, we should probably make it `readonly`.

Reply:
Make sense. Updated